### PR TITLE
Try setting up github actions

### PR DESCRIPTION
**Pull request by FreHu**

### CODE REVIEW

**Comment by larshp** [comment]
method `radians_to_degrees` running

**Comment by larshp** [comment]
method `degrees_to_radians` running

```
    "skip": [
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "initialization"},
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "goto_"},
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "pen_up_down"},
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "forward"},
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "back"}
    ]
```

**Comment by larshp** [comment]
all except two methods running,

```
    "skip": [
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "forward"},
      {"object": "ZCL_TURTLE", "class": "lcl_tests", "method": "back"}
    ]
```

File: src/core/zcl_turtle.clas.testclasses.abap [hunk 1]
```diff
@@ -62,7 +62,7 @@ class lcl_tests implementation.
     turtle->goto( x = 150 y = 0 ).
     turtle->forward( 100 ).
 
-    cl_abap_unit_assert=>assert_equals( exp = 150 + 100 act = turtle->position-x ).
+    cl_abap_unit_assert=>assert_equals( exp = 100 act = turtle->position-x ).
```
Comment (larshp) [inline src/core/zcl_turtle.clas.testclasses.abap]: can you help changing this back? its difficult to debug when the unit test is not as expected

**Comment by larshp** [comment]
is it safe to add round() here? new x and y are pixels anyhow?

https://github.com/FreHu/abap-turtle-graphics/blob/efac5eac5631497dda168049a3a41218f93e5aaf/src/core/zcl_turtle.clas.abap#L149-L150

**Comment by FreHu** [comment]
Should work as before, but it's not as easy as you may think, builtin function parameters are sometimes weird about what they accept
image

**Comment by larshp** [comment]
🎉 running, think the round() change can be rolled back if you like

**Comment by FreHu** [comment]
let's see

**Pull request merged**